syntax = "proto3";

package Map_renderer;

import "svg.proto";

message MapRenderer {
    double width = 1;
    double height = 2;
    double padding = 3;
    double line_width = 4;
    double stop_radius = 5;
    uint32 bus_label_font_size = 6;
    Svg.Point bus_label_offset = 7;
    uint32 stop_label_font_size = 8;
    Svg.Point stop_label_offset = 9;
    Svg.Color underlayer_color = 10;
    double underlayer_width = 11;
    repeated Svg.Color color_palette = 12;
}
